src/runtime_src/core/common/xdp/profile.cpp

```diff
@@ -548,14 +557,43 @@ update_device(void* handle, bool hw_context_flow)
   }
   
   if (xrt_core::config::get_aie_profile()) {
-    try {
-      xrt_core::xdp::aie::profile::load();
+    if (xrt_core::config::get_xdp_mode() == "xdna") {
+      xrt_core::message::send(xrt_core::message::severity_level::debug, "XRT",
+        "VE2 XDNA is set. Profiling will be available only for XDNA device.");
```

Comment:
Please update the message to say "xdp_mode" config is set to XDNA, instead of VE2 XDNA as we are not using the env variable anymore.